src/utils/validators.ts

```diff
@@ -30,11 +36,7 @@ export function ensureValidSetRequest(
   ttl: number
 ) {
   ensureValidKey(key);
-
-  if (!value) {
-    throw new InvalidArgumentError('value must not be empty');
-  }
-
+  ensureValidValue(value);
```

Comment:
This code was wrong. It should be possible to set a key and value of `''`. Don't carry that forward to the dictionary requests. It's also not necessary to check for nil, the types on the SimpleCacheClient methods prevent that.

I think ensureValidSetRequest should be inlined. It's [called in one place](https://github.com/momentohq/client-sdk-javascript/blob/a2eb281c17ccc32d03aca412b432d7b547d730a7/src/internal/cache-client.ts#L122).